docs/open-source-project/porting/system.md

```diff
@@ -100,6 +100,14 @@ To use most of the `resourced` functionalities, you must enable the following `c
 >
 > To use the `resourced` freezer feature, you must install the freezer plugin by enabling `CONFIG_FREEZER`.
 
+Resourced supports several swapping scenarios to provide more RAM to applications running in the foreground. To support them, enable the following options in the kernel configuration:
+
+  - `CONFIG_MEMCG_SWAP`: control swapping applications' memory, `CONFIG_SWAP` will be enabled automatically
+  - `CONFIG_ZRAM`: compressed block device
+  - `CONFIG_ZRAM_LZ4_COMPRESS`: an alternative compression algorithm, by default ZRAM uses LZO
+  - `CONFIG_ZSWAP`: compress memory before or dumping it to a storage device
```

Comment:
This sentence is not clear. Does it mean: 

Compresses memory before adding it to a storage device.